be/src/olap/data_dir.cpp

```diff
@@ -59,14 +59,14 @@ DataDir::DataDir(const std::string& path, int64_t capacity_bytes,
         TabletManager* tablet_manager, TxnManager* txn_manager)
         : _path(path),
         _capacity_bytes(capacity_bytes),
+        _available_bytes(0),
+        _disk_capacity_bytes(0),
```

Comment:
_capacity_bytes and _disk_capacity_bytes are redundant ?

Reply:
No, `_capacity_bytes` is user specified disk capacity, but it currently not used.
`_disk_capacity_bytes` is the capacity of the disk